.pre-commit-config.yaml

```diff
@@ -0,0 +1,42 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1  # Use the ref you want to point at
+    hooks:
+      - id: trailing-whitespace
+      - id: detect-aws-credentials
+      - id: check-symlinks
+      - id: check-yaml
+      - id: check-merge-conflict
+      - id: check-json
+      - id: check-docstring-first
+      - id: check-executables-have-shebangs
+      - id: check-builtin-literals
+      - id: check-ast
+-   repo: https://github.com/psf/black
+    rev: 21.9b0
+    hooks:
+      - id: black
+        language_version: python3.8
+-   repo: https://gitlab.com/pycqa/flake8
+    rev: 3.9.2
+    hooks:
+    - id: flake8
+-   repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.910
+    hooks:
+    -   id: mypy
+-   repo: https://github.com/pycqa/isort
+    rev: 5.8.0
+    hooks:
```

Comment:
@reiktar I have run into issues with isort and black compatibility in the past and have used one of the methods defined in this guide to avoid the compatibility issues:  https://pycqa.github.io/isort/docs/configuration/black_compatibility.html

Reply:
Done!